### DISCUSSION TITLE
Would like to code with someone :)

### DISCUSSION THREAD

**Discussion by M1sterGh0sT** [codewars/codewars.com#3248] [General]
Guys if someone wanna have fun and write some code lets keep in touch.

**Comment by Fares403** [comment]
Nice

**Reply by Abhijitdrag** [reply]
hello, i am new here. can someone tell me how is it work ?

**Reply by BioSam-Data** [reply]
Let's do it

**Reply by gaikiir** [reply]
am here guys we can work together 24/7 javascript, node js , react, laravel, vue , tailwindcss, php, flutter am there you hit me up on my github gaikiir you will find me there

**Comment by EchoqueenCC** [comment]


**Reply by Valentethedev** [reply]
I am here

**Reply by JurajkoDS** [reply]
How would pair programming work?:)

**Reply by Valentethedev** [reply]
Yes

**Comment by Valentethedev** [comment]
> Guys if someone wanna have fun and write some code lets keep in touch.

I am looking for a pair too

**Reply by dani12e** [reply]
Hey I'm new to programming though

**Reply by Valentethedev** [reply]
Do u have discord?

**Reply by dani12e** [reply]
yes i do

**Reply by Valentethedev** [reply]
Add me 
valenteartur#2805

**Reply by dani12e** [reply]
Sent a request

**Reply by bayness** [reply]
yes I would love to join, I work on Mern stack, and I feel little the hit the plato motivation, feeling bit demotivated, I'm down to join a group or another coder

**Comment by VanessaGreene** [comment]
Hi all, It's great to see you looking for more connections through Codewars. If you aren't on our Discord server yet, I invite you all [to join](https://discord.gg/7U9t33jrgG). Please feel free to use our posts section to discuss finding folks to pair with 🥰

**Comment by naol728** [comment]
i wanna code with someone to do some spacial project

**Reply by Valentethedev** [reply]
We can go
Learn together
🚀🚀🚀🚀

**Reply by Valentethedev** [reply]
My discord is valenteartur#2805

**Comment by klybovI** [comment]
I am learning python, I have a desire to grow, are there people willing to take me "under their wing"?😌
i`m from is Russia
telegram @klybov

**Comment by samuelnimi** [comment]
hi

**Comment by Merdas369** [comment]
> Guys if someone wanna have fun and write some code lets keep in touch.

ok , let's go.

**Comment by awaluddin-dev** [comment]
i'm just try codewars, so we can battle  here ?

**Comment by hobovsky** [comment]
This type of discussion is off-topic for GitHub. We handle social and learning matters on our [Discord server](https://discord.gg/mSwJWRvkHA) and its `#ask-help` and `#beginner-intros` channels, where it’s easier, and safer for everyone to join public channels and learn together.

It’s much more effective (and fun!) to share progress and ask questions in open channels with man users rather than private groups. Please continue this conversation on Discord instead.
